src/CBLReplicatorConfig.hh

```diff
@@ -195,6 +196,11 @@ namespace cbl_internal {
 
         // Writes a LiteCore replicator optionsDict
         void writeOptions(Encoder &enc) const {
+            static const unsigned kDefaultMaxRetriesOneShort = 9;
```

Comment:
Typo -- should be "...OneShot"

Reply:
I will remove all these to use LiteCore default value.